plugins/producer_plugin/include/eosio/producer_plugin/producer_plugin.hpp

```diff
@@ -44,6 +44,9 @@ class producer_plugin : public appbase::plugin<producer_plugin> {
 
    struct snapshot_information {
       chain::block_id_type head_block_id;
+      uint32_t             head_block_num;
```

Comment:
So the new fields are only added to snapshot_information of producer_plugin? At first I thought the snapshot format itself would be changed (which should be avoided).

Reply:
Correct, changes are confined to producer_plugin.